base.cfg

```diff
@@ -205,3 +205,4 @@ plone.schema = git git://github.com/plone/plone.schema.git pushurl=git@github.co
 
 [versions]
 plone.restapi =
+feedparser =
```

Comment:
@tiberiuichim added by accident?

Reply:
No, it won't build otherwise. There's a pinned version in upstream that breaks this.

Reply:
Do we have a feedparser dependency in the nav portlet code?!

Reply:
No, the dependency is in Plone.

https://github.com/plone/buildout.coredev/blob/20beb512d2931ddd43558fa1c3086b1e15d35eac/versions.cfg#L89

I no longer have access to my development computer, but looking at the feedparser readme, it [looks rough](https://pypi.org/project/feedparser/5.2.1/) with its python3 compatibility. I run ArchLinux and it always upgrades core packages to their latest released version (so, for example, I get to use python3.9 a lot sooner then other distros). So it might be a combination of those two factors.

Reply:
@tiberiuichim ok, this might be the reason why the Python 2.7 builds are all failing. I actually had troubles with Python 3.8 and 3.9 on OS X so I am currently using 3.7, so I can't really check on my machine right now.